tests/test_maintel_lasertracker_align.py

```diff
@@ -0,0 +1,139 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import asyncio
+import random
+import types
+import unittest
+
+from lsst.ts.idl.enums.LaserTracker import LaserStatus
+from lsst.ts.salobj import State
+
+from lsst.ts import standardscripts
+from lsst.ts.standardscripts.maintel.laser_tracker import Align, AlignComponent
+
+random.seed(47)  # for set_random_lsst_dds_partition_prefix
+
+
+class TestAlign(standardscripts.BaseScriptTestCase, unittest.IsolatedAsyncioTestCase):
+    async def basic_make_script(self, index):
+        self.script = Align(index=index, add_remotes=False)
+        self.script.mtcs = unittest.mock.AsyncMock()
+        self.script.mtcs.configure_mock(
+            **{
+                "offset_m2_hexapod": self.apply_offsets,
+                "offset_camera_hexapod": self.apply_offsets,
+            }
+        )
+
+        self.script.laser_tracker.rem.lasertracker_1 = unittest.mock.AsyncMock()
+        self.script.laser_tracker.rem.lasertracker_1.configure_mock(
+            **{
+                "cmd_align.set_start": self.mock_laser_align,
```

Comment:
`mock_laser_align` is not doing anything besides waiting 0.5 seconds.. I think you can remove it.